src/components/Swap/Summary/Summary.tsx

```diff
@@ -1,24 +1,30 @@
 import { useLingui } from '@lingui/react'
 import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 import { PriceImpact } from 'hooks/useUSDCPriceImpact'
-import { ArrowRight } from 'icons'
+import { ArrowDown, ArrowRight } from 'icons'
 import { PropsWithChildren } from 'react'
+import styled from 'styled-components/macro'
 import { ThemedText } from 'theme'
 import { formatCurrencyAmount } from 'utils/formatCurrencyAmount'
 
-import Column from '../../Column'
+import { default as Col } from '../../Column'
 import Row from '../../Row'
 import TokenImg from '../../TokenImg'
 
+const Column = styled(Col)<{ collapsedDetails: boolean }>`
```

Comment:
Rename this SummaryColumn and continue to use Column as the default import from ../../Column? Otherwise the names across files become more confusing.